charts/atlas-deployment/templates/atlas-deployment.yaml

```diff
@@ -24,9 +24,11 @@ spec:
   deploymentSpec:
   {{- toYaml .deploymentSpec | nindent 4 }}
   {{- end }}
+  {{- if .serverlessSpec }}
   serverlessSpec:
   {{- toYaml .serverlessSpec | nindent 4}}
   {{- end }}
```

Comment:
Where is the start for this end?

Reply:
Line 23 closes line 26 and line 27 closes line 30

Reply:
https://github.com/mongodb/helm-charts/pull/365/files#diff-19f72865b4e04ba2886de1959b7cbc8edbe2a3d30dba5761dd160ff1f50c9fa6R1

Reply:
I see, it is the range at the top

ok